elide-datastore/elide-datastore-hibernate5/src/test/java/com/yahoo/elide/datastores/hibernate5/HibernateDataStoreSupplier.java

```diff
@@ -24,6 +27,11 @@
 public class HibernateDataStoreSupplier implements Supplier<DataStore> {
     @Override
     public DataStore get() {
+        // Add additional checks to our static check mappings map.
+        // NOTE: This is a bit hacky. We need to do a major overhaul on our test architecture
+        TestCheckMappings.MAPPINGS.put("filterCheck", Filtered.FilterCheck.class);
```

Comment:
Isn't CriteriaCheck also deprecated?  So any CriteriaCheck in our examples should probably be replaced with a FilterExpressionCheck (which is not tightly coupled to Hibernate in any way).